machine.go

```diff
@@ -327,15 +324,12 @@ func NewMachine(ctx context.Context, cfg Config, opts ...Opt) (*Machine, error)
 
 	if m.logger == nil {
 		logger := log.New()
-		if cfg.Debug {
-			logger.SetLevel(log.DebugLevel)
-		}
 
 		m.logger = log.NewEntry(logger)
 	}
 
 	if m.client == nil {
-		m.client = NewClient(cfg.SocketPath, m.logger, cfg.Debug)
+		m.client = NewClient(cfg.SocketPath, m.logger, false)
```

Comment:
So the way of setting the flag is giving a new client?

Reply:
Yes